internal/postprocessor/main.go

```diff
@@ -122,14 +122,19 @@ type config struct {
 	googleapisDir  string
 	googleCloudDir string
 	modules        []string
-	branchPrefix   string
+	branchOverride string
 	githubUsername string
 	prFilepath     string
 	isOwlBotPR     bool
 }
 
 // isOwlBotPR uses git to tell if the PR being updated was authored by OwlBot.
-func isOwlBotPR(dir string) (bool, error) {
+func isOwlBotPR(dir, branchOverride string) (bool, error) {
```

Comment:
nit: should we name this something like runPostprocessor so that it encompasses an OwlBot run or a branchOverride run?

Reply:
Seems reasonable to me.